packages/nextjs/src/index.server.ts

```diff
@@ -79,6 +79,15 @@ export function init(options: NextjsOptions): void {
   const activeDomain = domain.active;
   domain.active = null;
 
+  if (typeof options.tunnel !== 'undefined') {
+    try {
```

Comment:
I would actually move this check into the `@sentry/node` SDK which the Next.js SDK depends on.

Reply:
This I'm now struggling with. Moving into the node `init` function causes the complexity to be above 20; I've currently created a helper function to get around the limitation.

Reply:
After moving to `@sentry/node` should the tests be duplicated there, or just completely moved?

Reply:
IMO the complexity rule is also something we can disable for that line. It's all just guidelines that shouldn't prevent us from shipping.

I think we should just move the tests to `@sentry/node` - no duplication needed.

Reply:
done